flex-config/ui_attributes.common.json

```diff
@@ -263,7 +263,72 @@
           "location": false,
           "agent_skills": true
         }
+      },
+      "queues_view_data_tiles": {
+        "enabled": true,
+        "activeTasksDataTile": false,
+        "waitingTasksDataTile": false,
+        "longestWaitTimeDataTile": false,
+        "agentsByActivityBarChart": false,
+        "channels": {
+          "voice": {
+            "color": "#ADD8E6",
+            "SLADataTile": true,
+            "taskCountsDataTile": true
+          },
+          "chat": {
+            "color": "#87CEFA",
+            "SLADataTile": false,
+            "taskCountsDataTile": false
+          },
+          "sms": {
+            "color": "#4682B4",
+            "SLADataTile": true,
+            "taskCountsDataTile": true
+          },
+          "video": {
+            "color": "#00CED1",
+            "SLADataTile": false,
+            "taskCountsDataTile": false
+          }
+        },
+        "allChannelsDataTile": true,
+        "enhancedAgentByActivityPieChart": true,
+        "agentActivityConfiguration": {
+          "Available": {
+            "color": "green",
+            "icon": "Accept"
+          },
+          "Outbound": {
+            "color": "greenyellow",
```

Comment:
This color is difficult to see the white text on, can we try another?

Reply:
Changed to darkgreen